.changesets/breaking_glasser_pq_metric_attribute_consistency.md

```diff
@@ -0,0 +1,9 @@
+### More consistent attributes on `apollo.router.operations.persisted_queries` metric ([PR #6403](https://github.com/apollographql/router/pull/6403))
+
+Version 1.28.1 added several *unstable* metrics, including `apollo.router.operations.persisted_queries`.
+
+When an operation is rejected, Router includes a `persisted_queries.safelist.rejected.unknown` attribute on the metric. Previously, this attribute had the value `true` if the operation is logged (via `log_unknown`), and `false` if the operation is not logged. (The attribute is not included at all if the operation is not rejected.) This appears to have been a mistake, as you can also tell whether it is logged via the `persisted_queries.logged` attribute.
+
+Router now only sets this attribute to true, and never to false. This may be a breaking change for your use of metrics; note that these metrics should be treated as unstable and may change in the future.
```

Comment:
I think saying it always sets it to true (you used the word "always" rather than "only") might be misleading — it's that *if* it is setting the attribute at all, the value is true, but it does not always set the attribute at all.

I did think it was good to explicitly note that these metrics were described as "unstable" when created — your edit doesn't use the word "unstable", and it might make people think that the decision to be OK with changing them is new in this release rather than carried over from the past. (On the other hand, "unstable" doesn't seem to be a term we use much in other contexts in Router CHANGELOG, so maybe that's why you're avoiding it?)

Reply:
I also did like saying explicitly that the old code was a mistake/error/bug/something, but maybe you don't.

Reply:
> it's that if it is setting the attribute at all, the value is true

ah, good point, will edit that to `... the router now only sets the ...` 


> these metrics were described as "unstable" when created 

can you clarify what it means for these GA (non-experimental, non-preview) metrics to be "unstable"? Would they change outside the norms of a GA or LTS release?

> saying explicitly that the old code was a mistake/error/bug/something

will edit the suggestion to `This however was unintentional and redundant with ...`

Reply:
I think it's ok to go with this. We're looking again at metrics stability for router 2.0